### DISCUSSION TITLE
Train a new speaker from existing multispeaker model

### DISCUSSION THREAD

**Discussion by jerichosiahaya** [coqui-ai/TTS#3068] [General Q&A]
Hey👋

I want to train a new speaker from an existing multispeaker model, I got the model from this gentleman here: https://github.com/Wikidepia/indonesian-tts

My question is, if I want to train it using new datasets with only one speaker, do I need to use single speaker mode or multi speaker mode when training? And, do I need to follow the config.json from that existing model?

I'm pretty sure I need to use multi speaker mode when train since the existing model has its own weights for the speakers, but I still need more explanation, and it's a bit difficult to find explanation here, I've tried to find answer.

Thank you.

**Comment by SophieDC98** [comment]
[NanoNomad](https://www.youtube.com/@nanonomad) has some interesting tutorials with colabs for training

**Reply by jerichosiahaya** [reply]
I've watched his videos, but he doesn't use the existing model with multispeakers. I tried to train using 100+ samples of data, but the result is gibberish. Do I need to add more data or is there any step that I missed?

**Reply by SophieDC98** [reply]
try to unfreeze your text encoder and DP

**Reply by jerichosiahaya** [reply]
By unfreezing, you mean adding this?
```python
config.model_args.freeze_DP=False
config.model_args.freeze_encoder=False
```
Before init the trainer?

**Reply by SophieDC98** [reply]
yep
